src/main/java/eu/europeana/metis/sandbox/entity/DatasetEntity.java

```diff
@@ -12,7 +12,7 @@
 
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
-  private String datasetId;
+  private Integer datasetId;
```

Comment:
In the database this field is of type serial and I rely on the DB to keep the sequence of it, until now mapping it to a String worked but as soon as I query for it, it broke because incompatible types, so I am keeping as a Integer here, but still return it as a String in the rest of the application